package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build": "turbo run build"
+    "build":  "for dir in packages/*; do (cd \"$dir\" && yarn); done && turbo run build"
```

Comment:
running `yarn install` on project root directory is the same as installing each package